command/status.go

```diff
@@ -18,13 +19,15 @@ func (c *StatusCommand) GetHelperText() string {
 	return "Returns the status of the Polygon SDK client"
 }
 
+func (c *StatusCommand) GetBaseCommand() string {
+	return "status"
+}
+
 // Help implements the cli.Command interface
 func (c *StatusCommand) Help() string {
 	c.Meta.DefineFlags()
 
-	usage := "status"
-
-	return c.GenerateHelp(c.Synopsis(), usage)
+	return types.GenerateHelp(c.Synopsis(), c.GetBaseCommand(), c.flagMap)
```

Comment:
You need to call `GenerateUsage` because of `--grpc-address` global flag.
```suggestion
	return types.GenerateHelp(c.Synopsis(), types.GenerateUsage(c.GetBaseCommand(), c.flagMap), c.flagMap)
```

Reply:
You're right - I missed this when adding in the generateUsage